hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/checksum/ContainerChecksumTreeManager.java

```diff
@@ -157,6 +159,28 @@ private void write(KeyValueContainerData data, ContainerProtos.ContainerChecksum
     }
   }
 
+  public byte[] readChecksumFileAsBytes(KeyValueContainerData data)  {
+    long containerID = data.getContainerID();
+    Lock readLock = getReadLock(containerID);
+    readLock.lock();
+    try {
+      File checksumFile = getContainerChecksumFile(data);
+
+      try (FileInputStream inStream = new FileInputStream(checksumFile)) {
+        return IOUtils.toByteArray(inStream);
```

Comment:
Can we use proto `ByteString` directly to reduce the number of copies required from 2 to 1? I think the only usage of this method requires a `ByteString` as output.